src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/prisonersearch/PrisonerSearchClient.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.client.prisonersearch
+
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import reactor.core.publisher.Mono
+
+class PrisonerSearchClient(private val prisonerSearchApiWebClient: WebClient) {
+
+  fun getPrisoner(prisonerNumber: String): Prisoner? =
+    prisonerSearchApiWebClient
+      .get()
+      .uri("/prisoner/{prisonerNumber}", prisonerNumber)
+      .retrieve()
+      .bodyToMono(Prisoner::class.java)
+      .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
+      .block()
+}
+
+// TODO add additional fields as and when needed e.g. ACTIVE/NOT ACTIVE in prison
+data class Prisoner(
```

Comment:
As per comment in the description ideally I would have generated this type but open API generator was not playing nicely.